public/index.html

```diff
@@ -99,15 +100,25 @@
             button:hover {
                 background: #218838;
             }
+            /* Contador de caracteres dentro do campo */
+            #charCount {
+                
+                
+                bottom: 5px;
+                font-size: 12px;
+                color: #aaa;
+            }
         </style>
     </head>
     <body>
         <h1>Blog de QTSW</h1>
 
         <form id="postForm">
-            <input type="text" id="content" placeholder="Digite sua postagem" required>
-            <button type="submit">Adicionar</button>
-        </form>
+        <div style="position: relative;">
+            <input type="text" id="content" placeholder="Digite sua postagem" required maxlength="100">
+            <span id="charCount">0/100 caracteres</span>
+        </div>
+        <button type="submit">Adicionar</button>
```

Comment:
A tag de fechamento do `form` me parece ter sido removida por acidente. Além disso, precisamos ajustar a identação do código.

```suggestion
            <div style="position: relative;">
                <input type="text" id="content" placeholder="Digite sua postagem" required maxlength="100">
                <span id="charCount">0/100 caracteres</span>
            </div>
            <button type="submit">Adicionar</button>
        </form>
```

Reply:
ajustei